services/alert/dao.go

```diff
@@ -1,5 +1,7 @@
 package alert
 
+//go:generate easyjson dao.go
```

Comment:
This is going to fail the CI build since the easyjson command does not exist. You can add the command to the Gopkg.toml requires list and then install it as an explicit step from the vendor dir during the build. See https://github.com/influxdata/kapacitor/blob/master/build.py#L154.